x/farming/keeper/reward.go

```diff
@@ -248,6 +278,7 @@ func (k Keeper) WithdrawRewards(ctx sdk.Context, farmerAcc sdk.AccAddress, staki
 		k.DecreaseOutstandingRewards(ctx, stakingCoinDenom, rewards)
 	}
 
+	k.decrementReferenceCount(ctx, stakingCoinDenom, staking.StartingEpoch-1)
 	staking.StartingEpoch = currentEpoch
 	k.SetStaking(ctx, stakingCoinDenom, farmerAcc, staking)
```

Comment:
@hallazzang Shouldn't we need to `k.incrementReferenceCount` as new `StartingEpoch`?